### ISSUE TITLE
[question] Is this plugin ready for the official plugin page?

### ISSUE THREAD

**Issue by peterk** [trajche/SamlAuth#10]
Is the plugin ready to be listed on the official plugin page? Found it by accident when looking for SAML support and I guess it would be easier if it was listed there.

**Comment by trajche** [comment]
@peterk Yeah it is, it's been on my mind but haven't done a pull request. Feel free to add it if you wish?

**Comment by peterk** [comment]
Have you tried against the latest version? If it still works I can try and do a patch for the info page.

**Comment by thomasDOTwtf** [comment]
@peterk I've tried it against the latest version is working after one fix I've already pull requested.
